services/core-web/src/components/common/RenderRadioButtons.js

```diff
@@ -23,13 +23,16 @@ const defaultProps = {
 
 const RenderRadioButtons = (props) => {
   const { meta, label, disabled, input, id, customOptions } = props;
-  const radioValue = typeof input.value !== "undefined" ? input.value.toString() : "";
 
   const options = customOptions ?? [
-    { label: "Yes", value: "true" },
-    { label: "No", value: "false" },
+    { label: "Yes", value: true },
+    { label: "No", value: false },
```

Comment:
Will this affect any of our backend endpoints that expect a boolean as a string or would this purely be a frontend thing? Looks like the be checks for booleans as a string in a couple of places:
![Uploading image.png…]()

Reply:
Good question, I will check.

Reply:
I think it should be fine. The examples in the screenshot are for checks that happen during filtering. The columns in the tables in the backend expect a boolean data type.
 
I also saw another ideal example in the explosives permit form. **closedOptions** is passed as an argument to customOptions parameter with boolean option values. 

**
![image](https://github.com/bcgov/mds/assets/72251620/daeffc62-2b86-4a2a-a296-ccede1316f47)
**

![image](https://github.com/bcgov/mds/assets/72251620/d66321da-54ee-486e-a151-34d3d1759827)